packages/nodes-base/nodes/Google/Sheet/v2/actions/sheet/update.operation.ts

```diff
@@ -398,17 +398,17 @@ export async function execute(
 				columnNames.concat([...newColumns]),
 			]);
 		} else {
-			preparedData = await sheet.prepareDataForUpdateOrUpsert(
-				data,
-				columnsToMatchOn[0],
+			preparedData = await sheet.prepareDataForUpdateOrUpsert({
+				inputData: data,
+				indexKey: columnsToMatchOn[0],
 				range,
-				headerRow,
-				firstDataRow,
+				keyRowIndex: headerRow,
+				dataStartRowIndex: firstDataRow,
```

Comment:
same as before could you add names for the rest of the parameters to make the code more consistent and readable?